### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by guru-sulakhe**

### CODE REVIEW

**Comment by guru-user** [comment]
why you added oil to karam dosa ? oil is already there

**Comment by guru-sulakhe** [comment]
> why you added oil to karam dosa ? oil is already there

for good taste it is neccessary to add extra oil to karma dosa

File: menu.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dosa 
+karam
```
Comment (guru-user) [inline menu.txt:2]: why karam is added to dosa

File: menu.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dosa
```
Comment (guru-user) [inline menu.txt:1]: why did you added dosa?

**Review by guru-user — APPROVED** [review]

**Pull request merged**